<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentForm</class>
 <widget class="QWidget" name="DepartmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>판매내역</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_code">
         <property name="text">
          <string>코드</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_code"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_price">
         <property name="text">
          <string>가격</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_price"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_saleCnt">
         <property name="text">
          <string>판매개수</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_saleCnt"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_marginRate">
         <property name="text">
          <string>마진율</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_marginRate"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>추가</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_update">
         <property name="text">
          <string>수정</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_del">
         <property name="text">
          <string>삭제</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btn_init">
         <property name="text">
          <string>초기화</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
